### ISSUE TITLE
responsive bug in chromium-based browsers

### ISSUE THREAD

**Issue by GerwarKi** [JiHong88/SunEditor#574]
Tested in reactjs with [suneditor-react](https://github.com/mkhstar/suneditor-react), but more likely to be a problem with stylesheets.
**App.js**
```javascript
import SunEditor from 'suneditor-react';
import './App.css';
import 'suneditor/dist/css/suneditor.min.css';
export default function App() {
  return (
    <div className="App">
	<SunEditor />
    </div>
  );
}
```
**App.css**
```css
body {
    background-color: orange;
}
```
Tested in
Google chrome (w10 & android): bug
Microsoft Edge (w10 & android): bug
Firefox (w10): Works fine

![issue](https://user-images.githubusercontent.com/31028951/101105451-33c5d580-3593-11eb-9a51-b13c7766cda4.gif)

**Comment by JiHong88** [comment]
@GerwarKi This bug seems to be a bug in the debugging console.

<img width="1330" alt="스크린샷 2021-01-11 오후 6 46 23" src="https://user-images.githubusercontent.com/13191953/104165986-6e961580-543d-11eb-95af-921e201ec8c7.png">

**Comment by GerwarKi** [comment]
I agree that it looks like a bug in the browser, but not only in debug mode, it happens even on small screen phones. In this example, it's too noticeable on my personal phone. I found it very strange and severe for a browser error, and more at this point without being noticed before.
